content/en/platform/corda/4.8/open-source/key-concepts-node.md

```diff
@@ -70,33 +69,32 @@ are documented in [API: RPC operations](api-rpc.md).
 Internally, the node has access to a rich set of services that are used during flow execution to coordinate ledger
 updates. The key services provided are:
 
-* Information on other nodes on the network and the services they offer
-* Access to the contents of the vault and the storage service
-* Access to, and generation of, the node’s public-private keypairs
-* Information about the node itself
-* The current time, as tracked by the node
+* Information on other nodes on the network and the services they offer.
```

Comment:
```suggestion
* Information about other nodes on the network and the services they offer.
```

Reply:
was this change committed? Looks like it was resolved and not applied?